text/0000-serialize-deserialize.md

```diff
@@ -0,0 +1,174 @@
+* Start Date: 2018-08-01
+* RFC PR: https://github.com/fusionjs/rfcs/pull/8
+
+# Summary
+
+This RFC describes a process for standardizing on an API for serializing and deserializing state between server and browser plugins.
+
+# Basic example
+
+```js
+import {SharedStateToken, createPlugin} from 'fusion-core';
+
+export default createPlugin({
+  deps: {
+    SharedState: SharedStateToken,
+  },
+  middleware: ({SharedState}) => {
+    const {serialize, deserialize} = SharedState('some-id');
+    return (ctx, next) => {
+      if (__BROWSER__) {
+        // load and parse serialized data from dom node
+        const data = deserialize();
```

Comment:
What do you do with the resulting data in this case?

How do you make it available to other parts of the app? (either redux state or react context)

Reply:
That would still be done via react context

Reply:
I'm referencing the fact that `data` is assigned to a variable but then nothing is done with it. Can you add examples for redux (I know you have one later down) and React context?